addon-test-support/fill-in.js

```diff
@@ -13,8 +14,19 @@ import wait from 'ember-test-helpers/wait';
 */
 export function fillIn(selector, text) {
   let el = getElementWithAssert(selector);
+
+  if (!isFocusable(el)) {
```

Comment:
unfortunately it's wrong. 
according to [isFocusable](https://github.com/cibernox/ember-native-dom-helpers/blob/6adbe755c5e0120ab9bfbf49631da75a8ffc3de0/addon-test-support/-private/is-focusable.js#L2) we treat `<a>` and `<link>` as a fillable elements now. 

I think the most straightforward solution is to extract `isContentEditable` and `isFormControl`(or something) helpers and reuse it in here in `fillIn`